types/superagent-prefix/index.d.ts

```diff
@@ -0,0 +1,9 @@
+// Type definitions for superagent-prefix 0.0
+// Project: https://github.com/johntron/superagent-prefix
+// Definitions by: Michael Ledin <https://github.com/mxl/>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.2
+
+import * as request from 'superagent';
+
+export default function(prefix: string): request.Plugin;
```

Comment:
@mhegazy I tried but build fails https://travis-ci.org/DefinitelyTyped/DefinitelyTyped/builds/239514755 with
> Module '"/home/travis/build/DefinitelyTyped/DefinitelyTyped/types/superagent-prefix/index"' resolves to a non-module entity and cannot be imported using this construct.

It seems that it doesn't work with functions - https://github.com/Microsoft/TypeScript/issues/5073. Any thoughts?